### PR TITLE
Draft Create Compliance with Safeguards PR

### PR DESCRIPTION
**Pull request by tbarnes94**
# Description

Added in --imagePullSecret flag to enable CRIP safeguard.  Reverted imageRegex for CAI safeguard to '.*' which allows all images through until we finalize that imageRegex (doesn't make sense to require one image type only at this point in time when a new user invokes `draft create`).  Draft Create -> manifest mode now compliant with safeguards.  Helm and Kustomize still need further tuning to be compliant due to the nature of those tools.

Fixes # (issue)
Feature # (details)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).

- Existing UTs as well as manual testing of `draft validate` against generated manifests from `draft create`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sallasindhuja — APPROVED** [review]

**Review by davidgamero — CHANGES_REQUESTED** [review]
lgtm

**Review by davidgamero — APPROVED** [review]
lgtm

File: template/deployments/manifests/manifests/deployment.yaml [hunk 1]
```diff
@@ -15,9 +15,23 @@ spec:
...
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - store
```
Comment (NickKeller) [inline template/deployments/manifests/manifests/deployment.yaml:26]: change `store` to `{{APPNAME}}`

Comment (tbarnes94) [reply template/deployments/manifests/manifests/deployment.yaml]: done

**Pull request closed without merging**